woocommerce/woocommerce
# Preserve dash in Mongolia postcodes

Preserves Mongolia dash when saved, previously, if you inserted a postcode with a dashcode, it gets removed.

This is the behavior for all postcodes except the ones we handle in `wc_format_postcode`, which are 16 countries.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. In Checkout block, use the following postcode when checking out with Mongolia `12345-1234`
2. Place the order, notice the postcode still has a dash in Order confirm screen.
3. Add a new item to cart, go back to Checkout, make sure the postcode is there, without an error, and with the dash.
4. Checkout with only `12345`. Make sure in order confirm screen that it doesn't have an extra dash.
5. Go back to Checkout with a new product, make sure it's not with an extra dash.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Preserve dash when saving Mongolia postcode.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
